1_fetch.R

```diff
@@ -450,7 +451,7 @@ p1_targets_list <- list(
     p1_drb_distance_matrix_updown,
     p1_drb_distance_matrix$updown
   ),
-  #save as csv for use in Python
+  #save as csv for readability
```

Comment:
I was wondering whether we still want to keep this csv file target, but I guess it doesn't hurt to keep as a way of quickly checking the format/dimensions of the updown matrix.